src/effects/VST/VSTEffect.cpp

```diff
@@ -952,7 +941,8 @@ std::shared_ptr<EffectInstance> VSTEffect::DoMakeInstance()
    GetConfig(*this, PluginSettings::Shared, wxT("Options"),
```

Comment:
If all is done right, `DoMakeInstance` and the `const_cast` should no longer be needed.
`mUseLatency` and `mUserBlockSize` don't need to be member variables of the effect.  They can become local variables of this function.  But then they must be passed to the constructo of Instance which stores them.

But this may need to wait until step 5.

Reply:
I now pass them to the Instance constructor. As you said, later in step 5. I could remove them from the effect.